test/parallel/test-fs-readfile-fd.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
-const common = require('../common');
+require('../common');
+const fixturesPath = require('../common/fixtures').path();
```

Comment:
This is close, but would be better as:

```js
const fixtures = require('../common/fixtures');
```

then ...

```js
const fn = fixtures.path('empty.txt');
```

Reply:
Okay!

Here is another one:
https://github.com/nodejs/node/pull/16112

Reply:
@tobbik, I'm going to close this one in favor of #16112 then. Just for future reference, you can update an existing PR by pushing changes to the corresponding branch on your GitHub.